tests/test_files/Model/Table/BarBarsAbstractTable.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace App\Model\Table;
+
+use Cake\ORM\Table;
+
+/**
+ * @property \App\Model\Table\FooTable&\Cake\ORM\Association\BelongsTo $Foo
+ * @property \Awesome\Model\Table\HousesTable&\Cake\ORM\Association\BelongsToMany $Houses
+ * @property \Awesome\Model\Table\WindowsTable&\Cake\ORM\Association\HasMany $Windows
+ *
+ * @method \App\Model\Entity\BarBarsAbstract get($primaryKey, $options = [])
+ * @method \App\Model\Entity\BarBarsAbstract newEntity($data = null, array $options = [])
+ * @method \App\Model\Entity\BarBarsAbstract[] newEntities(array $data, array $options = [])
+ * @method \App\Model\Entity\BarBarsAbstract|false save(\Cake\Datasource\EntityInterface $entity, $options = [])
+ * @method \App\Model\Entity\BarBarsAbstract saveOrFail(\Cake\Datasource\EntityInterface $entity, $options = [])
+ * @method \App\Model\Entity\BarBarsAbstract patchEntity(\Cake\Datasource\EntityInterface $entity, array $data, array $options = [])
+ * @method \App\Model\Entity\BarBarsAbstract[] patchEntities($entities, array $data, array $options = [])
+ * @method \App\Model\Entity\BarBarsAbstract findOrCreate($search, callable $callback = null, $options = [])
+ *
+ * @mixin \Cake\ORM\Behavior\TimestampBehavior
+ * @mixin \Tools\Model\Behavior\ConfirmableBehavior
+ * @mixin \MyNamespace\MyPlugin\Model\Behavior\MyBehavior
+ */
+class BarBarsAbstractTable extends AbstractTable {
+
+	/**
+	 * @param array $config
+	 * @return void
+	 */
+	public function initialize(array $config) {
```

Comment:
Proper indentation (tab) is off here.

Reply:
Not sure what exactly is wrong here?

Reply:
maybe github is confused.

but

...................................................S.......PHP Fatal error:  Cannot instantiate abstract class App\Model\Table\AbstractTable in /home/travis/build/dereuromark/cakephp-ide-helper/src/Annotator/ModelAnnotator.php on line 304

sounds like an actual issue.